services/ingestion/etl/just/csv/src/lib/transform.js

```diff
@@ -0,0 +1,166 @@
+// @flow
+
+import getCountryCode from '../../../../../helpers/getCountryCode';
+
+/*
+ * Transform message (JUST CSV)
+ */
+
+import type { Project } from '../../../../_types/Project';
+
+const getFundingArea = () => [];
+
+/**
+ * Preprocess locations
+ *
+ * Input fields taken from the `record` are:
+ *
+ * - `Project location longitude`
+ * - `Project location latitude`
+ * - `Project country`
+ *
+ * @memberof AgriCsvTransform
+ * @param {Object} record The row received from parsed file
+ * @returns {Array} List of {Location} objects for `project_locations` field
+ */
+const getLocations = record => {
+  const projectLongitude = record.field_prj_longitude || '';
```

Comment:
These fields are special, they are numbers:
```
/**
 * Describes field `project.project_locations.centroid`.
 * @type {Coordinates}
 */
type Coordinates = {
  lat: number,
  lon: number,
};
```
So, please default them to numbers or somehow avoid these castings, as below you are going to re-cast them to numbers anyway.